src/Packagist/WebBundle/Resources/config/algolia_settings.yml

```diff
@@ -1,22 +1,22 @@
 searchableAttributes:
-    - "name"
-    - "package_name"
-    - "package_organisation"
-    - "description"
-    - "tags"
+  - "unordered(package_name)"
+  - "unordered(tags)"
+  - "unordered(package_organisation)"
+  - "description"
```

Comment:
`description` should be `unordered` as well, no?

Reply:
Description is the whole `README.md` so I though ordered would make more sense. I will test and compare 👌

Reply:
@julienbourdeau description only contains the "description" field from composer.json. The readme content is not indexed at the moment. Maybe we should add it though.

Reply:
Oh my bad! I tested with ordered and unordered, I didn't find much differences on the tested queries. Although, since it's the composer.json description, I agree that unordered make more sense. I'll update.

Reply:
I think you didn't see any differences because of the `desc(popularity)` ranking criterion hijacking the ranking formula before `attributes`.